packages/react-instantsearch-core/src/core/createConnector.tsx

```diff
@@ -113,6 +113,13 @@ export function createConnectorWithoutContext(
             this.props
           );
         }
+
+        if (isWidget) {
+          const isServer = true; // TODO: find a real value for this
```

Comment:
this condition might be something related to `process`, but I'm not sure what bundle size impacts that will have